lib/teach-api.js

```diff
@@ -7,6 +7,11 @@ var ga = require('react-ga');
 
 var STORAGE_KEY = 'TEACH_API_LOGIN_INFO';
 
+// login emulation for local testing
```

Comment:
good point - yes, I added that in this PR first, then split it out, I guess technically it's not necessary for this pr since we're not faking logins, so I'll reset the teach-api.js lib in this PR instead and then #2018 can add in the login emulation